spec/oncocore.txt

```diff
@@ -199,13 +204,13 @@ Based on:		SurgicalProcedure
 Description:	"A surgical action addressing a cancer condition, independent of action context."
 // It is cancer-related only if the reason is a cancer condition
 0..1			ReasonReference value is type CancerCondition
-0..1			ReasonCode from CancerDisorderVS
+0..1			ReasonCode from AnyCancerDisorderCodeVS
```

Comment:
Not sure I understand the name change.

Reply:
It includes primary and secondary cancers.  The other sets include primary cancers and secondary cancers separately.